hack/installers/install-kubectx-linux.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I though kubectx is only for convenience. Why do we need to install it in e2e tests?